.github/workflows/weekly-serverless.yml

```diff
@@ -0,0 +1,34 @@
+name: Test Serverless Environment
+run-name: Creating Serverless Environment by @${{ github.actor }}
+
+on:
+    workflow_dispatch: # TODO: remove
+    schedule:
+      - cron: '0 0 * * 1' # every Monday at 00:00
+
+jobs:
+  naming:
+    runs-on: ubuntu-latest
+    outputs:
+      deployment_name: ${{ steps.set_deployment_name.outputs.date-name }}
+    steps:
+      - name: Set deployment name
+        id: set_deployment_name
+        run: |
+          echo "::set-output name=date-name::weekly-env-$(date +'%d-%b')"
+
+  deploy:
+    needs: naming
+    uses: ./.github/workflows/test-environment.yml
+    secrets: inherit
+    # Required for the 'Deploy' job in the 'test-environment.yml' to authenticate with Google Cloud (gcloud).
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    with:
+      deployment_name: ${{ needs.naming.outputs.deployment_name }}
+      ess-region: "region-1"
+      elk-stack-version: 8.12.0
```

Comment:
Do we need to bump this manually on every new release?

Reply:
No, I have to set it to 8.12 because of the bug https://github.com/elastic/cloudbeat/issues/1679